src/lib/Icons.ts

```diff
@@ -149,161 +147,154 @@ import { faUsersSlash as faUsersSlashSolid } from '@fortawesome/pro-solid-svg-ic
 import { faVenus as faVenusSolid } from '@fortawesome/pro-solid-svg-icons/faVenus'
 
 type IconName =
-  | 'achievement'
-  | 'add'
-  | 'add-user'
-  | 'advance'
+  | 'badge-check' // achievement
```

Comment:
Ik heb de oorspronkelijke namen nog even laten staan als comment. Dat is wel handig als we ObsIdentify ook doen. Kan daarna weg.